From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Refrag <R3FR4GYT@gmail.com>
Date: Sun, 4 Feb 2024 17:45:18 +0100
Subject: [PATCH] i_system.c hacks

This patch disables / removes some code to get the game to build without issues.
These don't seem to matter much anyway as the game still runs.
---
 src/sdl/i_system.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/sdl/i_system.c b/src/sdl/i_system.c
index b2396e269b8d7cc456ee9f525aeda7991835cd4e..54977b6ba144375ec89200b6596f6e56f2a7c340 100644
--- a/src/sdl/i_system.c
+++ b/src/sdl/i_system.c
@@ -84,14 +84,11 @@ typedef LPVOID (WINAPI *p_MapViewOfFile) (HANDLE, DWORD, DWORD, DWORD, SIZE_T);
 #else
 #include <sys/statvfs.h>
 #include <sys/param.h>
-#include <sys/mount.h>
 /*For meminfo*/
 #include <sys/types.h>
 #ifdef FREEBSD
 #include <kvm.h>
 #endif
-#include <nlist.h>
-#include <sys/sysctl.h>
 #endif
 #endif
 
@@ -2103,6 +2100,8 @@ void I_StartupMouse2(void)
 			else
 				rts = 1;
 		}
+	
+		#if 0
 		if (dtr != -1 || rts != -1)
 		{
 			INT32 c;
@@ -2119,6 +2118,7 @@ void I_StartupMouse2(void)
 				c |= TIOCM_RTS;
 			ioctl(fdmouse2, TIOCMSET, &c);
 		}
+		#endif
 	}
 	mouse2_started = 1;
 	I_AddExitFunc(I_ShutdownMouse2);
